packages/components/src/components/Modal/Modal.module.css

```
@@ -11,13 +11,31 @@
   flex-direction: column;
   min-width: 200px;
   padding: var(--dds-modal-container-padding);
+  padding-top: 0;
 }
 
 .container-scrollable {
   max-height: calc(100vh - var(--dds-spacing-x1));
   overflow: auto;
 }
 
+.modal-header {
+  display: flex;
+  align-items: center;
+  padding-top: var(--dds-spacing-x1);
+  padding-bottom: var(--dds-spacing-x1);
```

Spacing på bunnen skal være litt mindre:
```suggestion
  padding-bottom: var(--dds-spacing-x0-75);
```